### PR TITLE
Added fetching items from server

### PR DESCRIPTION
**Pull request by unspectd**

### CODE REVIEW

File: pkg/store/models.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package store
 import "time"
 
 type Item struct {
-	Id        int
-	Name      string
-	CreatedAt time.Time `db:"created_at"`
+	Id        int `json:"id"`
+	Name      string `json:"name"`
+	CreatedAt time.Time `db:"created_at" json:"created_at"`
```
Comment (evleria) [inline pkg/store/models.go]: `json:"created_at"` => `json:"createdAt"`

File: web/src/state/appReducer.ts [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
-  items: ReadonlyArray<ItemEntity>;
-}>;
+export interface IAppState {
+  items: ItemEntity[];
+}
 
-export type AppAction = ActionType<typeof actions>;
-
-const initialState: AppState = {
+const initialState: IAppState = {
```
Comment (evleria) [inline web/src/state/appReducer.ts]: rename back

**Review by evleria — CHANGES_REQUESTED** [review]

File: pkg/store/models.go (hunk 1 above)
Comment (unspectd) [reply pkg/store/models.go]: done

File: web/src/state/appReducer.ts (hunk 2 above)
Comment (unspectd) [reply web/src/state/appReducer.ts]: done

**Pull request merged**